### ISSUE TITLE
[IDEA] Seo Features: page title and meta tags 

### ISSUE THREAD

**Issue by par7133** [TiddlyWiki/TiddlyWiki5#5977]
Hello,

I'm a little frustrated I can't set anything among page title, keywords and other meta tags of the page.

Daniele Bonini
5 Mode

**Comment by pmario** [comment]
Hi Daniele, 
Issues, should be actionable. So they should start with something like this. 

"As a user interested in SEO I would like to .......  because ...." 

That will give us the possibility, to know what you would like to change and why. At the moment we know that you are a little frustrated.

**Comment by giffmex** [comment]
@pmario @par7133 I think Daniele's request is clear, though it probably belongs in the Discourse forum (or Google Groups, considering when this was posted). She wants to know how to edit the HTML page title and keywords and meta tags for a TiddlyWiki file. She would like instruction on how to do that. Had she asked on Discourse/Google group, she would have got her answer. Hopefully she did. If she added it here, she probably would also like an easy way to set those somewhere in the control panel.

**Comment by twMat** [comment]
@giffmex - you say the poster "would like instruction on how to do that" but it could be a feature request of some kind... or, actually, multiple requests, which is also a bit of a crime when posting issues. That said, I'd love to see OP discussed in the boards.

I think this should be closed. (Who can close posts here anyway? Only Big J?)

**Comment by giffmex** [comment]
@twMat Yes, like I said, if she is requesting a feature, it is most likely that she wants an easy way to edit the HTML page title, keywords and meta tags. And the control panel would be the proper place for that. I am just guessing she was a new user and did not know the ins and outs of posting issues on Github vs the forums. The forum would have been better for a question, but probably she was probably also asking for that functionality to be more accessible.
